nextcord/types/auto_moderation.py

```diff
@@ -18,6 +18,7 @@ class AutoModerationTriggerMetadata(TypedDict, total=False):
     presets: List[KeywordPresetType]
     allow_list: List[str]
     mention_total_limit: Optional[int]
+    mention_raid_protection_enabled: bool
```

Comment:
`nextcord.AutoModerationTriggerMetadata` needs to be updated to handle this attribute.